### PR TITLE
Bugfix 20

### PR DESCRIPTION
**Pull request by christopherdancy**

### CODE REVIEW

File: src/web3/transactions.ts [hunk 1]
```diff
@@ -26,23 +26,24 @@ export const useTransaction = () => {
...
           toast.dismiss(toastId);
           if (txReceipt.status === 0) {
             toast(failedMessage);
+            if (stoppedCallback) stoppedCallback();
           } else if (txReceipt.status === 1) {
             toast(successMessage);
             if (waitingCallback) waitingCallback();
           } else {
             toast(failedMessage);
+            if (stoppedCallback) stoppedCallback();
```
Comment (adamgall) [inline src/web3/transactions.ts:42]: Please spend some more time in this function.

Is this `else` block ever going to get hit? I don't think so, according to the docs here: https://docs.ethers.io/v5/api/providers/types/#providers-TransactionReceipt

What concept is being handled with this variable name `stopped`? I'm not sure what "stopped" means in the context of a transcation.

Comment (christopherdancy) [reply src/web3/transactions.ts:42]: I still need to go through and clean the entire transaction page up. My suggestion is this should be another PR.

Comment (adamgall) [reply src/web3/transactions.ts:42]: Agreed

Comment (christopherdancy) [reply src/web3/transactions.ts:42]: Will create this now

Comment (adamgall) [reply src/web3/transactions.ts:42]: @christopherdancy https://github.com/fractal-framework/fractal-interface/issues/33

**Review by adamgall — APPROVED** [review]
Approving with the understanding that the `useTransaction` hook will be cleaned up, tracking here: https://github.com/fractal-framework/fractal-interface/issues/33

File: src/web3/transactions.ts (hunk 1 above)
Comment (christopherdancy) [reply src/web3/transactions.ts:42]: Oh cool - will delete the one I created

**Pull request merged**